### PR TITLE
Add AM2R to daily generate

### PR DESCRIPTION
**Pull request by Miepee**
grafik
am2r is relatively quick to generate, so would that like on two days
bumped the time to be earlier to hopefully be less conflicting with other schedules (and also because it aligns better with my timezone)

### CODE REVIEW

**Review by dyceron — CHANGES_REQUESTED** [review]
You're missing the section under `steps` for a2mr

**Comment by Miepee** [comment]
oops

**Comment by Miepee** [comment]
Would also need an orga/discord mod to add `AM2R_DEV_TALK_WEBHOOK` to the repo/server

File: .github/workflows/daily-generate.yml [hunk 1]
```diff
@@ -12,13 +12,15 @@ on:
...
           - samus_returns
+          - am2r
 
   schedule:
     - cron: '0 16 * * 3'  # Cave Story
     - cron: '0 23 * * *'  # Dread
     - cron: '0 16 * * 2'  # Prime 1
     - cron: '0 16 * * 1'  # Prime 2
     - cron: '0 16 * * 4'  # Samus Returns
+    - cron: '0 10 * * 1,5' # AM2R
```
Comment (dyceron) [inline .github/workflows/daily-generate.yml:23]: Is 1,5 going to cause issues? I know it's a regional thing but I am curious.

Comment (Miepee) [reply .github/workflows/daily-generate.yml:23]: See the image posted in the PR description :)

**Review by dyceron — APPROVED** [review]

**Pull request merged**